onboard/catkin_ws/src/offboard_comms/scripts/port_finder.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 
 sudo dmesg | \
-grep tty | \
-sed -n -e 's/^.* ch341-uart converter now attached to \([^ ]*\).*/\1/p' | \
-tail -n 1 | \
+grep "USB ACM device" | \
+awk '{print $5;}' | \
+sed 's/.$//' | \
```

Comment:
awk is not the best here, since if there is an extra space or something then it will mess up, sed should be better since its just pattern matching, so try using
```suggestion
grep tty | \
sed -n -e 's/^.* \([^ ]*\): USB ACM device/\1/p' | \
tail -n 1 | \
```